Source/x11/XGServerWindow.m

```diff
@@ -3190,7 +3192,7 @@ - (void) movewindow: (NSPoint)loc : (int)win
     }
 
   window->siz_hints.x = (int)loc.x;
-  window->siz_hints.y = (int)([self boundsForScreen: window->screen].size.height -
+  window->siz_hints.y = (int)([self boundsForScreen: window->monitor_id].size.height -
```

Comment:
The implementation here and in the next method needs to match _XFrameToOSFrame:, currently it does but it may be that all these are wrong. If the screen is bigger than the current monitor, this results in wrong window placements.

Reply:
Thanks, fixed.